gamemodes/terrortown/gamemode/client/cl_keys.lua

```diff
@@ -174,6 +175,13 @@ function GM:KeyPress(ply, key)
 		end
 
 		timer.Simple(0.05, _func)
+	elseif key == IN_FORWARD or key == IN_BACK or key == IN_MOVERIGHT or key == IN_MOVELEFT then
+		UpdateInputSprintFunc = UpdateInputSprintFunc or UpdateInputSprint
+
+		-- Might not be loaded yet due to load order
```

Comment:
What load order? isn't this deterministic as we define the order?

Reply:
Move this line up to the other shared includes, and it should work:
https://github.com/TTT-2/TTT2/blob/5c51c24077282f9b898f93ecff3125fcfb99f2a2/gamemodes/terrortown/gamemode/client/cl_main.lua#L74

Reply:
The cl_main load order looks pretty messy to me and looks like it will eventually be cleaned up at which point it would immediately break again. I will move it up nonetheless, but should I keep the check, just in case?

Reply:
I dont think it will be cleaned up soon, but yes please move it up and remove the checks, as they are useless and we want to let it fail quickly, when the loading order is incorrect. Otherwise we might not notice the bug as fast as possible.